articles/openshift/oshift-how-add-domains-proxy-whitelist.md

```diff
@@ -0,0 +1,122 @@
+---
+title: How to permit outbound access to internet hosts in deployments with public services network connectivity | UKCloud Ltd
+description: Outlines methods to securely permit access to internet hosts in v3.11 OpenShift clusters deployed with public services network connectivity.
+services: openshift
+author: Ben Bacon
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Permit outbound internet access in deployments with public services network connectivity
+toc_fullpath: How To/oshift-how-add-domains-proxy-whitelist.md
+toc_mdlink: oshift-how-add-domains-proxy-whitelist.md
+---
+
+# How to permit outbound access to internet hosts in deployments with public services network connectivity
+
+## Overview
+
+This article outlines the necessary steps to whitelist domain names within Squid proxy. Squid is an enabled service within v3.11 OpenShift clusters deployed with connectivity to public sector networks (such as: HSCN, PSN-A, Janet). Whitelisted domains can be accessed on the Internet via the proxy enabling you to request external resources/data on nodes that previously only had connectivity to public sector networks.
+
+## Prerequisites
+
+To complete the steps in this guide, you must have access to a cluster which is running OpenShift Container Platform v3.11 (or newer) with connectivity to a public services network. Squid proxy is not currently enabled within environments that do not have access to a public services network.
+
+You must also have access to either:
+
+- the OpenShift Web console
+- `oc`, the OpenShift command-line client (CLI). For more information, see OpenShift's [*Get Started with the CLI*](https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html)
+
+## Destination domain whitelist
+
+Squid proxy is installed as a service on the control plane load balancers. Within the Squid configuration, we have specified an ACL (Access Control List) as being a list of destination domains (present in a file on the filesystem) which herein we will refer to as the whitelist. Outbound traffic from the cluster's internal network will pass through the proxy with the destination domain being compared against the whitelist for each connection attempt; if the domain exists then outbound Internet traffic is permitted, otherwise it is denied with a HTTP 403 error.
```

Comment:
Make file system two separate words
I think you need hereafter rather that herein, or use simpler language, such as "which we will refer to as the whitelist for the rest of this article". (put a comma before the which in either case)
Lowercase i for internet

Reply:
Should it not be uppercase I seeing as it is a noun?

Reply:
Company standard is to use a lowercase i for internet